### PR TITLE
[DC-2352] [DC-2356] Add --run_as, --pmi_email, and --deid_questionnaire_response_map_dataset params to deid_runner.sh

### PR DESCRIPTION
**Pull request by hiro-mishima**

### CODE REVIEW

File: data_steward/tools/deid_runner.sh [hunk 1]
```diff
@@ -101,8 +111,12 @@ python "${TOOLS_DIR}/run_deid.py" --idataset "${cdr_id}" --private_key "${key_fi
...
 
+# clear GOOGLE_APPLICATION_CREDENTIALS environment variable inorder to make impersonation work
+unset GOOGLE_APPLICATION_CREDENTIALS
+gcloud config set account "${pmi_email}"
+
 # apply de-identification rules on registered tier dataset 
-python "${CLEANER_DIR}/clean_cdr.py" --project_id "${APP_ID}" --dataset_id "${registered_cdr_deid}" --sandbox_dataset_id "${registered_cdr_deid_sandbox}" --data_stage ${data_stage} --mapping_dataset_id "${cdr_id}"  --cope_lookup_dataset_id "${cope_lookup_dataset_id}" --cope_table_name "${cope_table_name}" -s 2>&1 | tee registered_tier_cleaning_log.txt
+python "${CLEANER_DIR}/clean_cdr.py" --project_id "${APP_ID}" --dataset_id "${registered_cdr_deid}" --run_as "${run_as}" --sandbox_dataset_id "${registered_cdr_deid_sandbox}" --data_stage ${data_stage} --mapping_dataset_id "${cdr_id}"  --cope_lookup_dataset_id "${cope_lookup_dataset_id}" --cope_table_name "${cope_table_name}" --deid_questionnaire_response_map_dataset "2022q2r1_rdr_sandbox" -s 2>&1 | tee registered_tier_cleaning_log.txt
```
Comment (jp3477) [inline data_steward/tools/deid_runner.sh:119]: The `deid_questionnaire_response_map_dataset` argument is hardcoded here. Should it be a variable from a shell script command line argument?

File: data_steward/tools/deid_runner.sh [hunk 2]
```diff
@@ -7,6 +7,8 @@ USAGE="
 Usage: deid_runner.sh
   --key_file <path to key file>
   --cdr_id <combined_dataset name>
+  --run_as <service account email for impersonation>
+  --pmi_email <pmi-ops account email>
```
Comment (jp3477) [inline data_steward/tools/deid_runner.sh:11]: Should `deid_questionnaire_response_map_dataset` be added here as an argument?

**Review by jp3477 — CHANGES_REQUESTED** [review]
@hiro-mishima Left some comments below. Let me know if my understanding is correct.

**Comment by hiro-mishima** [comment]
Thanks, @jp3477 for reviewing! I updated the script. It would be great if you could have a look at it again. Thanks!

**Review by ksdkalluri — APPROVED** [review]
We tested the changes in the last run, Good to merge.

**Pull request merged**